src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -557,24 +557,19 @@ const ActiveRecordingsToolbar: React.FunctionComponent<ActiveRecordingsToolbarPr
   const [warningModalOpen, setWarningModalOpen] = React.useState(false);
   const [actionToggleOpen, setActionToggleOpen] = React.useState(false);
 
-  const deletionDialogsEnabled = React.useMemo(
-    () => context.settings.deletionDialogsEnabledFor(DeleteOrDisableWarningType.DeleteActiveRecordings),
-    [context.settings]
-  );
-
   const handleActionToggle = React.useCallback(() => setActionToggleOpen((old) => !old), [setActionToggleOpen]);
 
   const handleWarningModalClose = React.useCallback(() => {
     setWarningModalOpen(false);
   }, [setWarningModalOpen]);
 
   const handleDeleteButton = React.useCallback(() => {
-    if (deletionDialogsEnabled) {
```

Comment:
For some reason the deletionDialogsEnabled memo was not being changed whenever a user clicked on "Do Not ask again", but using the context.settings directly works correctly.

Reply:
`context.settings.deletionDialogsEnabledFor` is not an observable rite? its pulling directly from localstorage and depends only on `context.settings` (never changes). Then, it makes sense that is not updating :D 

The latest changes can `context.settings.deletionDialogsEnabledFor` on -demand so it works ^^

Reply:
Ah, whoops yeah that makes sense.